### PR TITLE
.github: add actions to build for multiple arch

### PR DESCRIPTION
**Pull request by fjmolinas**
To work this PR will require that the following secrets are configured:

DOCKERHUB_USERNAME
DOCKERHUB_TOKEN

### CODE REVIEW

**Comment by fjmolinas** [comment]
@aabadie

File: .github/workflows/image.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+name: build
```
Comment (aabadie) [inline .github/workflows/image.yml:1]:
```suggestion
name: docker
```

Comment (aabadie) [reply .github/workflows/image.yml:1]: And rename the file docker.yml

**Comment by fjmolinas** [comment]
Addressed comments and added armv7

**Review by aabadie — APPROVED** [review]
The Dockerhub secrets are in place. Let's see if that works.

**Comment by aabadie** [comment]
It worked ! https://hub.docker.com/repository/docker/aabadie/ota-server/tags?page=1&ordering=last_updated
That's awesome. Thanks @fjmolinas !

**Pull request merged**